packages/commonwealth/client/scripts/controllers/server/sessions.ts

```diff
@@ -94,29 +93,34 @@ export async function signSessionWithMagic(
     signerAddress,
   );
 
+  const from = () => {
+    if (walletChain === ChainBase.CosmosSDK) return `magic:${signerAddress}`;
```

Comment:
this feels a little clunky, but I need to find a way to flag magic so it can be verified separately